### PR TITLE
[Bugfix:Submission] Fix submit as student/bulk upload button

### PR DESCRIPTION
**Pull request by DefinitiveAbove**
### What is the current behavior?
Using the `Make Submission for a Student` or `Bulk PDF Upload` buttons do not work and throw an error.
image

### What is the new behavior?
Fixes submit as student/bulk upload buttons; issue was caused when `submitty_ocr` feature flag is disabled.

### Other information?
Can test by disabling debug mode and the `submitty_ocr` feature flag (for debug, edit `config/database.json` at `/usr/local/submitty/config/database.json` and set `debugging_enabled` to false); make sure that the elements with ids `toggle-id-scan`/`use-ocr` do not exist on the submission page.

### CODE REVIEW

**Review by shailpatels — APPROVED** [review]
Thanks for making a quick fix for this!

File: site/ts/instructor-submission.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ function init(){
     const qrPrefixInput = document.getElementById('qr_prefix') as HTMLInputElement;
     const qrSuffixInput = document.getElementById('qr_suffix') as HTMLInputElement;
     const useQRCheckBox = document.getElementById('use-qr') as HTMLInputElement;
-    const useScanIdsCheckBox = document.getElementById('use-ocr') as HTMLInputElement;
+    const useScanIdsCheckBox = document.getElementById('use-ocr') as HTMLInputElement | null;
```
Comment (MasterOdin) [inline site/ts/instructor-submission.ts:43]: This change is essentially the same as removing the type assertion altogether and just having:
```
    const useScanIdsCheckBox = document.getElementById('use-ocr');
```

Comment (DefinitiveAbove) [reply site/ts/instructor-submission.ts:43]: document.getElementById provides `HTMLElement | null` - HTMLInputElement provides some additional properties such as the `checked` property so just having `document.getElementById('use-ocr')` throws an error with the checker.

Comment (MasterOdin) [reply site/ts/instructor-submission.ts:43]: Fair

**Pull request merged**